### ISSUE TITLE
Tablebase support

### ISSUE THREAD

**Issue by oscardssmith** [romstad/Chess.jl#7]
Are there plans to allow reading of `syzygy` or `gaviota` TBs from this library? Imo, if that was added, this would be at feature parity with Python Chess.

**Comment by romstad** [comment]
Yes, syzygy tablebase support is planned, but I don't know how soon it will be done.

**Comment by markus7800** [comment]
A while back I translated the python-chess syzygy implementation to Chess.jl.
It is not thoroughly tested because in the end I did not use it in my engine but it should be somewhat working.
My [code](https://github.com/markus7800/Syzygy.jl/blob/main/src/Syzygy.jl) could be used as starting point for the Chess.jl package.
